virtcontainers/pkg/cloud-hypervisor/Makefile

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+all: clean-generated-code
```

Comment:
Did you add this Makefile, and then do a make all in order to pull the rest?  Perhaps you can clarify what part you added versus what was created from the OpenAPI tooling?  It'll help me better understand how you carried this out / reproduce from scratch.  It *may* make sense to have that as two commits, to make it clear.  Otherwise, at least detail it out on the commit message for "clh: Generate client code"